src/index.js

```diff
@@ -727,22 +727,13 @@ function isObject(a) {
 }
 
 function isDocumentVisible() {
-  if (typeof document.visibilityState !== 'undefined') {
-    return (
-      document.visibilityState === 'visible' ||
-      document.visibilityState === 'prerender'
-    )
-  }
-
-  return true
+  return document.visibilityState === undefined
+    || document.visibilityState === 'visible'
+    || document.visibilityState === 'prerender'
```

Comment:
The changes here are not logically equivalent

Reply:
they are equivalent. see this [CodeSandbox](https://codesandbox.io/s/stupefied-http-l3v0p) console output

Reply:
Ah. Right you are!

Reply:
maybe some unit tests would be in order 😉